src/file-upload/file-item.styles.tsx

```diff
@@ -125,8 +175,7 @@ export const LoadingActionContainer = styled(ItemActionContainer)`
     }
 `;
 
-// TODO: Replace with new IconButton when ready
-export const IconButton = styled(Button.Small)`
+export const IconButton = styled(DSIconButton)`
```

Comment:
no need to set height/width any more, can specify the `sizeType` prop as small

Reply:
ah ok. i think because i started this before IconButton was updated. Thanks for pointing